app/org/sagebionetworks/bridge/validators/StudyValidator.java

```diff
@@ -42,9 +67,14 @@ public void validate(Object obj, Errors errors) {
             }
         }
         validateEmails(errors, study.getSupportEmail(), "supportEmail");
+        validateEmails(errors, study.getTechnicalEmail(), "technicalEmail");
         validateEmails(errors, study.getConsentNotificationEmail(), "consentNotificationEmail");
     }
     
+    private boolean isInRange(int value, int min) {
+        return (value >= min && value <= 100);
+    }
```

Comment:
Is there a reason 100 is a hardcoded constant while min is an input? Do we have plans to make the min vary from study, so the legacy studies can have a backwards compatible min of 2 and new studies have a more reasonable min (at least 8 if not 12).